src/renderer/components/+custom-resources/crd-details.tsx

```diff
@@ -143,11 +144,12 @@ export class CRDDetails extends React.Component<Props> {
         {validation &&
         <>
           <DrawerTitle title="Validation"/>
-          <AceEditor
-            mode="yaml"
-            className="validation"
+          <MonacoEditor
+            options = {{ automaticLayout: true, minimap: {enabled: false}, readOnly: true}}
```

Comment:
Would it be better to have a single place where the default options object is specified? Because this one doesn't turn off line numbers but the one in `add-cluster.tsx` does.